from services.errors.base import BaseServiceError

# TESTED ON 2024-10-14 05:27 GMT
class DatasetNameDuplicateError(BaseServiceError):
    pass

# TESTED ON 2024-10-14 05:27 GMT
class DatasetInUseError(BaseServiceError):
    pass
